go.mod

```diff
@@ -23,6 +23,7 @@ require (
 	github.com/safchain/ethtool v0.4.1
 	github.com/vishvananda/netlink v1.2.1-beta.2
 	golang.org/x/sys v0.21.0
+	sigs.k8s.io/knftables v0.0.16
```

Comment:
you. just cut 0.0.17 two days ago https://github.com/kubernetes-sigs/knftables/releases/tag/v0.0.17 , but it does not seem required here

Reply:
yeah, I did mean to update that, but then I forgot, and as you say, it doesn't really matter